### DISCUSSION TITLE
how can i curved text in fabric js

### DISCUSSION THREAD

**Discussion by virtualb2b** [fabricjs/fabric.js#9685] [General]
im new in Fabric JS i need function like this curved up and down please check attachment 
![arch-text](https://github.com/fabricjs/fabric.js/assets/96039307/ff8d9608-e1d3-48e0-8d61-dc48398c6910)
any one help me out

**Comment by ShaMan123** [comment]
http://fabricjs.com/text-on-paths

`new FabricText('ARCH TEXT', {path: new Path('...')})`

**Reply by virtualb2b** [reply]
dear that link you shear with me its now help full i just need same as curved up with double stroke fabric js curved text not support double stroke
